GVRf/Framework/framework/src/main/res/raw/hud_console.fsh

```diff
@@ -1,29 +1,15 @@
-// Copyright 2015 Samsung Electronics Co., LTD
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
- 
-
 precision mediump float;
```

Comment:
do not remove the copyright.  if yo want, you can update it to the current year.